bin/index.js

```diff
@@ -259,6 +260,18 @@ function checkIfUsesTypeSelector(nodeContext, node) {
   return "no error";
 }
 
+/*
+    All imports are disallowed except for main.css
+*/
+function checkIfHasImports(fileName, node) {
+  if (fileName !== "main" && node.type === "Atrule" && node.name === "import") {
+    return `  ${colors.underline(`on line ${node.loc.start.line}:`)}
+  There is an import rule.
+  All imports are disallowed except for main.css`;
```

Comment:
This could be phrased differently to also explain why this should be done this way.

> Imports are only allowed in main.css.
> Having all imports in one file guarantees that there is only one place in the project to see which CSS files are loaded. It also improves the site’s performance, since the browser only needs to load one file to know which files it needs to load afterwards.